### DISCUSSION TITLE
Is commit message character limit supported?

### DISCUSSION THREAD

**Discussion by asier-paz** [extrawurst/gitui#2271] [General]
Hello everyone.

I see there's a useful red text telling you the subject of a commit is greater than 50 characters but I can't seem to find anything for the commit message.

Is commit message character limit supported?

Thanks.

**Comment by extrawurst** [comment]
is there a "official" limit?

**Reply by asier-paz** [reply]
Well. Not "official". But kind of an extended good practice.

I don't think the limit of 50 characters in the subject is "official" either. As far as I know there's nothing official, but some extended good practices.

For example the 50/72. Although Linus advocates for 72-character columns for word-wrapping, except for quoted material that has a specific line format. ([see link](https://stackoverflow.com/a/17792445))

Maybe these settings should be configurable.
